### PR TITLE
The React capstone project

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
## The React capstone project
- You will select an API that provides numeric data about a topic that you like and then build the webapp around it. The webapp will have several pages: 👍🏽 
- one page with a list of items that could be filtered by some parameters 👍🏽
- one page for the item details 👍🏽  
- APIs used  [openweathermap]('api.openweathermap.org')  and  [restCountries](''https://restcountries.com/v3.1')

Warm Regards
@kinginthenorthcodez

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,91 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Stackoverflow!!!
+- Inspiration.
+- etc.
```
Comment (OLIPLICHE) [inline README.md:87]: - [ ] Please gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1,70 +1,91 @@
...
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Live Demo
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[video demo]('https://www.loom.com/share/084836369797419eb3f51274a58183b9');
+https://www.loom.com/share/084836369797419eb3f51274a58183b9
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[Live website]('');
```
Comment (OLIPLICHE) [inline README.md:29]: - [ ] Please deploy properly the application using Heroku, Netlify, or GitHub Pages.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Linter is green (the last commit have a green check mark (✅)) 👏
README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
The App is working fine 🥇 



## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
@OLIPLICHE  Required changes made 👍🏽

File: README.md [hunk 3]
```diff
@@ -1,70 +1,91 @@
...
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Live Demo
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[video demo]('https://www.loom.com/share/be2e9372ee88437f9c5102640a317be7');
+https://www.loom.com/share/be2e9372ee88437f9c5102640a317be7
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[Live website]('https://kinginthenorthcodez.github.io/air-pollution-metric-app');
```
Comment (DammyShittu) [inline README.md:29]: - [ ] Your demo link is not displayed on the README. You can fix that by removing the quotation marks, like this:

  ```
  [Live website](https://kinginthenorthcodez.github.io/air-pollution-metric-app);
  ```
  king-demo

- [ ] **[OPTIONAL]** I would recommend you move this link above the `video demo` presentation, just after the **Live Demo** heading for better visibility.

- [ ] In addition, your live demo does not really display anything. Kindly fix that.

  king-live

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (kinginthenorthcodez) [reply README.md:29]: @DammyShittu  Suggested changes made 👍🏽 

https://www.loom.com/share/bdc3bb63bd6d4dabbcd1a7883e701b5e

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @kinginthenorthcodez👋, 

### You did a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Good commit history. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
@thecodechaser Thank you for your kind compliment 👍🏽

**Comment by kinginthenorthcodez** [comment]
Merged #5  into main

**Pull request merged**